src/Blocks/BlockBigFlower.h

```diff
@@ -49,10 +49,6 @@ class cBlockBigFlowerHandler final :
 
 	virtual cItems ConvertToPickups(const NIBBLETYPE a_BlockMeta, const cItem * const a_Tool) const override
 	{
-		if (IsMetaTopPart(a_BlockMeta))
```

Comment:
The top half of big flowers don't store any flower type metadata, so I suspect with these changes punching the top half no longer gives the right drops.